step-functions/get-ivq-questions.asl.json

```diff
@@ -172,20 +138,20 @@
         }
       },
       "ResultPath": "$.result",
-      "Next": "increment index by one"
+      "Next": "Increment index by one"
     },
-    "increment index by one": {
+    "Increment index by one": {
       "Type": "Pass",
       "Parameters": {
         "count.$": "$.iterator.count",
         "currentQuestion.$": "$.iterator.currentQuestion",
         "correlationId.$": "$.iterator.correlationId",
         "index.$": "States.MathAdd($.iterator.index, 1)"
       },
-      "Next": "continue looping?",
+      "Next": "Should we continue looping?",
```

Comment:
I don't like the `we` that's why I removed it. Its a state machine? any this is no biggy

Reply:
I'll make this change